Zend/tests/list_001.phpt

```diff
@@ -5,6 +5,8 @@
 
 list($a, list($b)) = array(new stdclass, array(new stdclass));
 var_dump($a, $b);
+[$a, [$b]] = array(new stdclass, array(new stdclass));
+var_dump($a, $b);
```

Comment:
Oh, yeah, `list()` still doesn't support assignment by-reference. I'd wanted to fix that.